app/views/shipit/stacks/show.html.erb

```diff
@@ -6,7 +6,7 @@
 <div class="wrapper">
   <section>
     <header class="section-header">
-      <h2>Undeployed Commits</h2>
+      <h2>Undeployed PRs</h2>
```

Comment:
Its is actually commits. Sure most of the time you'll see merge commits from PRs, and they'll be deployed like this. But if someone push straight on master it won't be a PR.

Reply:
`Undeployed PRs/Commits` kind of :trollface: but not joking 🤷‍♂️

Reply:
A PR is a commit. Can you explain why you think we need to make distinction here?

Reply:
It's about consistency, as this page is using PRs and Commits as they are the same thing, and IMHO they are different.

Most of the time a PR has a single commit, but you can find a PR containing multiple commits.

The `Undeployed Commits` list talks about PR names and not about commits. 
<img alt="image" src="https://screenshot.click/2018-08-21_14-16-19_0yfjv.png">

If you go into the PRs you'll find the commit messages are different:

<img alt="image" src="https://user-images.githubusercontent.com/950087/44421026-7cd1f700-a54d-11e8-92c0-6b7b3d6a1a4c.png"  width="50%">

<img alt="image" src="https://user-images.githubusercontent.com/950087/44421034-82c7d800-a54d-11e8-9663-980e6f278e14.png" width="50%">


That said, Shopify core is using the merge PR action, that will create an extra commit, and this seems it might be what this is actually talking about.

On the above case a `PR is a commit` **if and only if** the merge PR action is used.